autoreporter_addons/ssl_checks/reporter.py

```diff
@@ -148,7 +148,11 @@ def create_reports(task_result: Dict[str, Any], language: Language) -> List[Repo
             # If the domain starts with www. but the version without www. is in the names list,
             # let's assume just the version without www. is advertised and used by the users.
             if not (payload["domain"].startswith("www.") and payload["domain"][4:] in result["names"]):
-                names_string = ", ".join(sorted(set(result["names"])))
+                names_list = sorted(set(result["names"]))
```

Comment:
`names_list = sorted(set(result["names"]))`
`names_string = ", ".join(names_list[:ExtraModulesConfig.MAX_CERTIFICATE_NAMES_TO_SHOW]`

Reply:
nope, if theres more we want to have dots